src/e2e-test/java/io.cdap.plugin/mssql/locators/MssqlLocators.java

```diff
@@ -14,38 +14,25 @@
  * the License.
  */
 
-package io.cdap.plugin.locators;
+package io.cdap.plugin.mssql.locators;
 
 import io.cdap.e2e.utils.SeleniumDriver;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 import org.openqa.selenium.support.How;
+
 /**
- * Oracle Plugin Locators.
+ * Mssql Plugin Locators.
  */
-public class ReplicationLocators {
-    @FindBy(how = How.XPATH, using = "//*[contains(text(),'Next')]")
-    public static WebElement next;
-    @FindBy(how = How.XPATH, using = "//div[contains(text(),'Oracle')]")
-    public static WebElement oraclePlugin;
+public class MssqlLocators {
+
     public static WebElement selectTable(String tableName) {
```

Comment:
Can we make this one locator common for all 3 plugins ?

Reply:
I have already made common to use for all the 3 DB plugins, but need to mention in a single file so to use from there. So can i mention in this file and reuse for other plugins?